.eslintrc.yaml

```diff
@@ -4,15 +4,15 @@ env:
 ignorePatterns:
   - dist
 
-plugins:
-  - jest
-
 extends:
   - plugin:@foxglove/base
+  - plugin:@foxglove/jest
 
 overrides:
   - files: ["*.ts", "*.tsx"]
     extends:
       - plugin:@foxglove/typescript
     parserOptions:
       project: ./tsconfig.json
+    rules:
+      "@foxglove/no-boolean-parameters": off
```

Comment:
why?

Reply:
This rule prevents us from implementing APIs that follow ES6 standard library conventions